scripts/itkdev-docker-compose

```diff
@@ -47,6 +49,15 @@ Commands:
   sql:port
                 Display the exposed MariaDB SQL server port.
 
+  traefik:start
+                Start træfik reserver proxy.
+
+  traefik:stop
+                Stop træfik reverser proxy
+
+  traefik:open
```

Comment:
For consistency with other similar commands, we should add `traefik:url` to the mix.

Reply:
Added